indico/modules/categories/controllers/base.py

```diff
@@ -44,6 +49,11 @@ def _checkParams(self):
 
 
 class RHDisplayCategoryBase(RHCategoryBase):
+
+    def __init__(self):
+        RHCategoryBase.__init__(self)
+        self.now = datetime.now(DisplayTZ().getDisplayTZ(as_timezone=True))
```

Comment:
Copy Event.display_tzinfo to Category; then you can use `now_utc().astimezone(self.category.display_tzinfo)`